src/System.Threading.ThreadPool/tests/ThreadPoolTests.cs

```diff
@@ -30,10 +30,13 @@ public static void ConcurrentInitializeTest()
                 {
                     countdownEvent.Signal();
                     countdownEvent.Wait(ThreadTestHelpers.UnexpectedTimeoutMilliseconds);
-                    Assert.True(ThreadPool.SetMinThreads(processorCount, processorCount));
                 };
 
+            // ThreadPool.SetMinThreads returns false on UapAot
```

Comment:
The test is testing concurrent calls to SetMinThreads (and thread pool initialization as part of that, based on CoreCLR implementation). In hindsight, the CoreCLR repo may be a better location for this test. In any case, for now if there's no issue with the test I would suggest keeping it as it was.